src/Components/Weather/Weather.css

```diff
@@ -91,7 +91,8 @@
 
 .loading {
   text-align: center;
-  font-size: 18px;
+  font-size: 3rem;
+  color: #000;
```

Comment:
- [ ] Kindly make sure your design match with the template. kindly check the screencast below to check more about this

https://drive.google.com/file/d/1TrGdiVB9qDIBjOeE-VWC82ueqfu9BJPP/view?usp=sharing